spec/working_hours/computation_spec.rb

```diff
@@ -757,7 +757,6 @@
     it 'uses precise computation to avoid useless loops' do
       # +200 usec on each time, using floating point would cause
       # precision issues and require several iterations
-      expect(self).to receive(:advance_to_working_time).twice.and_call_original
```

Comment:
I was getting this 
![image](https://user-images.githubusercontent.com/803765/146350456-58361ebb-7f28-4af2-a972-7efc752cce3b.png)

Not sure this tested anything useful anyway ?

Reply:
Mm well AFAIR according to the comment, it is testing something because previously it was executed a lot more times because of lack of precision. I just tried running this (including the removed line) on ruby 3.0.3 and AS 7 but I don't get the error you had. Can you still reproduce it? which versions are you testing on?

Reply:
Indeed, I overlooked that 🙈 

I pushed back this line to see if it fails on travis, I have still the issue locally

Reply:
Oh well it passes 🤷 I was testing on 3.0.0p0 so maybe that's the reason

Reply:
Ok great, thanks :)